File: Sdk-ObjectiveC/office365-lists-sdk/office365-lists-sdk/ListEntity.m
//
//  ListEntity.m
//  office365-lists-sdk
//
//  Created by Gustavo on 7/22/14.
//  Copyright (c) 2014 Lagash. All rights reserved.
//

#import "ListEntity.h"

@implementation ListEntity

- (BaseEntity *)createFromJson:(NSDictionary *)data{
    
    NSDictionary* metadata = [data valueForKey : @"__metadata"];
    
    [super createMetadata : metadata];
    
    self.Id = [data valueForKey : @"Id"];
    self.Title = [data valueForKey : @"Title"];
    
    return self;
}

@end

File: Sdk/office365_list_sdk/office365_list_sdk/ListEntity.swift
//
//  ListEntity.swift
//  office365_list_sdk
//
//  Created by Gustavo on 7/15/14.
//  Copyright (c) 2014 Lagash. All rights reserved.
//

import Foundation
import office365_base_sdk

class ListEntity: BaseEntity{
    
    var Title : NSString? = "";
    
    /*func setMetadata(data : NSDictionary){
        self.Metadata.Id = data.valueForKey("id") as String;
        self.Metadata.Uri = data.valueForKey("uri") as String;
        self.Metadata.type = data.valueForKey("type") as String;
    }*/
    
    override func createFromJson(data : NSDictionary) -> BaseEntity{
        
       // var metadata = data.valueForKey("__metadata") as NSDictionary;
       // self.setMetadata(metadata);
        
        self.Id = data.valueForKey("Id") as String;
        self.Title = data.valueForKey("Title") as String;
        
        return self;
    }
}